packages/types/src/user/service.ts

```diff
@@ -5,18 +5,23 @@ import {
   UpdateUserDTO,
 } from "./mutations"
 import { FilterableUserProps, InviteDTO, UserDTO } from "./common"
+import { RestoreReturn, SoftDeleteReturn } from "../dal"
 
 import { Context } from "../shared-context"
 import { FindConfig } from "../common"
 import { IModuleService } from "../modules-sdk"
-import { RestoreReturn, SoftDeleteReturn } from "../dal"
 
 export interface IUserModuleService extends IModuleService {
   validateInviteToken(
     token: string,
     sharedContext?: Context
   ): Promise<InviteDTO>
 
+  resendInvites(
+    inviteIds: string[],
+    sharedContext?: Context
+  ): Promise<InviteDTO[]>
+
```

Comment:
thought: not sure that this should live in the user module. 

"Sending" an invite is an opinionated way of sharing an invite token with someone. But the UserModule should not really care or even know about the possible methods of transfer between the application and the one needing the invite. 

I would instead move all of the logic related to "sending" to a workflow that simply retrieves the token from the user module and pushes it to whatever "sending" logic has been configured.

Reply:
We could still have a function in the UserService that can refresh an invite token, but what it is being used for afterwards should not be the module's concern.

Reply:
When reading the implementation this seems to be what is happening so maybe it's just a question of renaming this method to `refreshInvite`